packages/ui/Measurable/Measurable.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react/sort-prop-types */
```

Comment:
Is this linting directive still required? I wouldn't expect so given the code.

Reply:
@Djspaceg can you respond to this please?

Reply:
This looks like cruft from wherever I copied this starting-point code. I'll fix it up now.